Src/BootCamp.Chapter/Checks.cs

```diff
@@ -16,26 +16,37 @@ public static class Checks
     {
         public static int PromptInt(string message)
         {
-            // To do: call your implementation. 
-            return 0;
+            var Age = message;
```

Comment:
use consistent variable names. For local variables- camelCase (starts with non-capital)